edb/pgsql/inheritance.py

```diff
@@ -44,22 +44,26 @@ def get_inheritance_view(
                 or ptr_stor_info.table_type == 'ObjectType'
             ):
                 ptrs[ptrname] = (
-                    ptr_stor_info.column_name,
+                    [ptr_stor_info.column_name],
                     ptr_stor_info.column_type,
                 )
 
+                shortname = ptr.get_shortname(schema).name
+                if shortname != ptr_stor_info.column_name:
+                    ptrs[ptrname][0].append(common.quote_ident(shortname))
```

Comment:
Am I right in that this makes it such that each pointer is represented by two columns in the view?

One named after ptr's uuid and one using ptr shortname?
Well, except things like `id`, `source` and `target` which have only shortnames and no uuids.

We should add a comment explaining this. Or make the ptrs dict contain better types (datastructs with proper names).